brokerapi/brokers/stackdriver_trace/broker.go

```diff
@@ -56,7 +48,7 @@ func (b *StackdriverTraceBroker) Bind(instanceID, bindingID string, details mode
 	details.Parameters["role"] = "cloudtrace.agent"
 
 	// Create account
-	newBinding, err := b.ServiceAccountManager.CreateAccountInGoogle(instanceID, bindingID, details, models.ServiceInstanceDetails{})
+	newBinding, err := b.AccountManager.CreateAccountInGoogle(instanceID, bindingID, details, models.ServiceInstanceDetails{})
```

Comment:
now that it's AccountManager there's nothing to indicate ServiceAccount. Google account is pretty generic. Suggest renaming method to something like CreateServiceAccount